src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: bold;
```

Comment:
if you specify `font-weight` for the body tag, then all child tags will inherit this rule